ember-accounts/app/templates/components/x-history.hbs

```diff
@@ -0,0 +1,46 @@
+<div class="receipt-information">
+  <h3>Receipt info <small>(optional)</small></h3>
+  <p>
+  HuBoard does not provide invoices, but if you need additional contact or tax information added to your
+  receipts (business address, VAT number, etc.), enter it below and it will appear on all of your receipts.
+  </p>
+  <p>
+  {{ textarea value=model.history.additional_info }}
+
+  <button class="hb-button small" {{action 'saveAdditionalInfo' model }} disabled={{processing}} >Update</button>
+  </p>
+
+</div>
+
+ <div class="table-wrapper">
+  <table class="table">
+    <thead>
+      <th>Date</th>
+      <th>Account</th>
+      <th>Amount</th>
+      <th></th>
+    </thead>
+    <tbody>
+
+      {{#each model.history.charges as |charge|}}
+      <tr>
+        <td>{{stripe-date charge.created }}</td>
+        <td>{{ charge.receipt_email }}</td>
+        <td>{{stripe-money charge.amount }}</td>
+        <td class="text-right">    
+         {{#if charge.paid}}
+          <a href="/settings/invoices/{{unbound charge.invoice }}">HTML</a> | <a
```

Comment:
Does not seem to be in scope for this PR, but I made a mental note that HTML invoices would ideally have a link back to the receipt list.